### ISSUE TITLE
남은 기능

### ISSUE THREAD

**Issue by tipifv** [KNU-2022S-OSP-03-09/project#5]
### 매우 종요로움

- [ ] 예약받기(use.html/app.py)
- [ ] 찬 겨를(예약된 시간) 보여주기(use.html/app.py/common.py)

### 종요로움

- [ ] User 받기(use.html/app.py)
- [ ] 꾸미기(base.html/root.html)

### 겨를 남으면

- [ ] User 예약 보기/지우기(app.py/database.py/user.html)
- [ ] 지난 예약 지우기(app.py/database.py)

뭐하고 계신지 적어주세요.
저는 common.py에서 찬 겨를 셈하는 조각 만들고 있습니다.

**Comment by tipifv** [comment]
@wook8321 @PARKYUNJU @hwangsunkyeong

**Comment by PARKYUNJU** [comment]
예약받기를 진행해보고 있습니다

**Comment by PARKYUNJU** [comment]
현재 master에서 pull 해서 실행할 때 success.html 에 아무것도 없네요. 제 pyj 브랜치에 있는 success.html 올려도 될까요

**Comment by tipifv** [comment]
네 올리시거나 #3 을 뭉치세요(merge). 밑은(원래는) 기능이 다 되면 뭉치려 했는데 그런것까지 마음쓰다가는 다 못만들겠네요.

**Comment by wook8321** [comment]
저는 유저 받는 거 하겠습니다

**Comment by hwangsunkyeong** [comment]
base 꾸밀게요!!

**Comment by tipifv** [comment]
User 받기라고 했을때 제가 생각한건 use.html에서 예약할때 이름 학번 같이 적는쯤이고 로그인까지는 아니었는데
wook8321님이 어떻게 하고계신지는 모르겠네요

**Comment by PARKYUNJU** [comment]
제가 착각해서 이름이랑 학번 적는건 간단하게 만들었는데 common.py에 유저 클래스를 만들어서 저장해야 할까요?

**Comment by PARKYUNJU** [comment]
지금 master에 있는거 pull 해서 run 해보니까 room에서 use넘어가는 곳에서 에러가 생기네요
![20220609_035036](https://user-images.githubusercontent.com/86733856/172694040-b7d5f416-da85-4b9e-bd3e-360ca116fc13.png)

app.py를 손봐야 될까요

**Comment by tipifv** [comment]
아니요 쓰는이(유저)는 따로 풀그림(프로그램) 안에 갊지(저장하지) 않습니다. 데이터베이스에 Use 만들때 학번 이름 같이 넣으면 되는데 그 함수는 곧 올리겠습니다.

**Comment by tipifv** [comment]
@PARKYUNJU 님 쓰는이 받기 만드셨으면 가지에 올려주세요

**Comment by PARKYUNJU** [comment]
제 브랜치로 push가 안돼서 git pull 다시하고 master에 올렸습니다 git log 했을 떄 HEAD->master 잘뜨고
![image](https://user-images.githubusercontent.com/86733856/172769179-9c99dece-8925-4bbc-aeb9-659d7cee5506.png)

이런식으로 뜹니다

**Comment by PARKYUNJU** [comment]
오늘 몇시에 어디서모일까요?

**Comment by wook8321** [comment]
저 그럼 예약받기를 만들까요? app.py에  use 고치면 되나요?

**Comment by PARKYUNJU** [comment]
꾸미기 체크했습니다
